test/flux_calculator_tests.jl

```diff
@@ -50,3 +63,222 @@ end
+get_surface_params(::TestAtmos) = sf_params
+
+function get_thermo_params(sim::TestAtmos)
+    FT = sim.params.FT
+    aliases = string.(fieldnames(TP.ThermodynamicsParameters))
+    toml_dict = CP.create_toml_dict(FT; dict_type = "alias")
+    pairs = CP.get_parameter_values!(toml_dict, aliases, "Thermodynamics")
+    TP.ThermodynamicsParameters{FT}(; pairs...)
+end
+
+Interfacer.get_field(sim::TestAtmos, ::Val{:height_int}) = sim.integrator.p.z
+Interfacer.get_field(sim::TestAtmos, ::Val{:height_sfc}) = sim.integrator.p.z_sfc
+Interfacer.get_field(sim::TestAtmos, ::Val{:uv_int}) = @. StaticArrays.SVector(sim.integrator.p.u, sim.integrator.p.v)
+Interfacer.get_field(sim::TestAtmos, ::Val{:thermo_state_int}) =
+    TD.PhaseEquil_ρTq.(get_thermo_params(sim), sim.integrator.ρ, sim.integrator.T, sim.integrator.q)
+Interfacer.get_field(sim::TestAtmos, ::Val{:air_density}) = sim.integrator.ρ
+Interfacer.get_field(sim::TestAtmos, ::Val{:air_temperature}) = sim.integrator.T
+
+function update_sim!(sim::TestAtmos, fields, _)
+    (; F_turb_ρτxz, F_turb_energy, F_turb_moisture) = fields
+    ρ_int = sim.integrator.ρ
+    @. sim.integrator.p.energy_bc = -(F_turb_energy)
+    @. sim.integrator.p.ρq_tot_bc = -F_turb_moisture
+    @. sim.integrator.p.uₕ_bc = -(F_turb_ρτxz / ρ_int) # x-compoennt only for this test
+end
+
+# ocean sim object and extensions
+struct TestOcean{M, Y, D, I} <: Interfacer.SurfaceModelSimulation
+    model::M
+    Y_init::Y
+    domain::D
+    integrator::I
+end
+Interfacer.get_field(sim::TestOcean, ::Val{:surface_temperature}) = sim.integrator.T
+Interfacer.get_field(sim::TestOcean, ::Val{:air_humidity}) = sim.integrator.p.q
+Interfacer.get_field(sim::TestOcean, ::Val{:roughness_momentum}) = sim.integrator.p.z0m
+Interfacer.get_field(sim::TestOcean, ::Val{:roughness_buoyancy}) = sim.integrator.p.z0b
+Interfacer.get_field(sim::TestOcean, ::Val{:beta}) = sim.integrator.p.beta
+Interfacer.get_field(sim::TestOcean, ::Val{:area_fraction}) = sim.integrator.p.area_fraction
+Interfacer.get_field(sim::TestOcean, ::Val{:heat_transfer_coefficient}) = sim.integrator.p.Ch
+Interfacer.get_field(sim::TestOcean, ::Val{:drag_coefficient}) = sim.integrator.p.Cd
+Interfacer.get_field(sim::TestOcean, ::Val{:albedo}) = sim.integrator.p.α
+
+function surface_thermo_state(
+    sim::TestOcean,
+    thermo_params::TD.Parameters.ThermodynamicsParameters,
+    thermo_state_int,
+    colidx::Fields.ColumnIndex,
+)
+    T_sfc = Interfacer.get_field(sim, Val(:surface_temperature), colidx)
+    ρ_sfc = thermo_state_int.ρ # arbitrary
+    q_sfc = Interfacer.get_field(sim, Val(:air_humidity), colidx) # read from cache
+    @. TD.PhaseEquil_ρTq.(thermo_params, ρ_sfc, T_sfc, q_sfc)
+end
+
+function update_turbulent_fluxes_point!(sim::TestOcean, fields::NamedTuple, colidx::Fields.ColumnIndex)
+    (; F_turb_energy) = fields
+    (; F_turb_energy) = fields
+    @. sim.integrator.p.F_aero[colidx] = F_turb_energy
+end
+
+@testset "calculate correct fluxes: dry" begin
```

Comment:
Is the `correct` maybe redundant here?

Reply:
Well, it implies that not only it's performing the calculation, but that we're checking for the correct values of it using the analytical solution, so I'd prefer keeping it.